client/web/src/tracking/TelemetricRedirect.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback, useState } from 'react'
+
+import { Redirect } from 'react-router'
+
+import { LoaderButton } from '../components/LoaderButton'
+import { logEventSynchronously } from '../user/settings/backend'
+
+export interface TelemetricRedirectProps {
+    to: string
+    eventName: string
+    children: React.ReactNode
+    className?: string
+}
+
+const MaximumTelemetryDelay = 5000
```

Comment:
We have sorta a convention to name global constants upper case 

```suggestion
const MAXIMUM_TELEMETRY_DELAY = 5000
```